testing/test_package/lib/override_class.dart

```diff
@@ -0,0 +1,13 @@
+import 'base_class.dart';
+
+class BoxConstraints extends Constraints {
+  /// Creates box constraints with the given constraints.
+  const BoxConstraints(
```

Comment:
weird formatting here. Shouldn't this end up being `();`?

Reply:
yes, formatted.